sdk/storage/azure-storage-files-shares/inc/azure/storage/files/shares/rest_client.hpp

```diff
@@ -31,7 +31,7 @@ namespace Azure { namespace Storage { namespace Files { namespace Shares {
     /**
      * The version used for the operations to Azure storage services.
      */
-    constexpr static const char* ApiVersion = "2022-11-02";
+    constexpr static const char* ApiVersion = "2023-01-03";
```

Comment:
This API version doesn't seem to be consistent with the Api Version being used in rest_client.cpp. That seems ... unexpected.

Reply:
I've checked. The API version in File Share is `2023-01-03`. The version in [sdk/storage/azure-storage-files-shares/inc/azure/storage/files/shares/rest_client.hpp](https://github.com/Azure/azure-sdk-for-cpp/pull/4612/files/47d954925bc9881e41099205c7349cd07388209e#diff-dbca2f58a39f741a856131a0af52ff1c85d04fefeb104eaa87cb9050c4646ba9) and [sdk/storage/azure-storage-files-shares/src/rest_client.cpp](https://github.com/Azure/azure-sdk-for-cpp/pull/4612/files/47d954925bc9881e41099205c7349cd07388209e#diff-538706fea4818b4beb3e685f885c3dc9223f232a56ca69b0d1db57002b4bd524) is the same.

The API version in Blobs is `2022-11-02`. The version in [sdk/storage/azure-storage-blobs/inc/azure/storage//blobs/rest_client.hpp](https://github.com/Azure/azure-sdk-for-cpp/pull/4612/files/47d954925bc9881e41099205c7349cd07388209e#diff-497cf142755af575a9dae45d23cbab05ffd91f4ca22c4c8a5e21258385155066) and [sdk/storage/azure-storage-blobs/src/rest_client.cpp](https://github.com/Azure/azure-sdk-for-cpp/pull/4612/files/47d954925bc9881e41099205c7349cd07388209e#diff-d048cafffda39a598fbe5978c65b9ac139089d2321e18c4591a6920c19b6ab53) is the same.